ui/app/src/utils/content.ts

```diff
@@ -466,13 +466,23 @@ function parseElementByContentType(
             // TODO: Groovy Controller Issue;
             path = getInnerHtml(item.querySelector(':scope > key'));
           }
-          const instance = parseContentXML(
-            component ? wrapElementInAuxDocument(component) : null,
-            path,
-            contentTypesLookup,
-            instanceLookup
-          );
-          array.push(instance);
+          const itemKey = getInnerHtml(item.querySelector(':scope > key'));
```

Comment:
```ts
const itemKey = getInnerHtml(item.querySelector(':scope > key'));
if (!path && !component) {
  path = itemKey
}
```
(You can take out that TODO; it doesn't really say much and this sort of addresses)